mappings/net/minecraft/client/options/GraphicsOption.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_5365 net/minecraft/client/options/GraphicsOption
```

Comment:
I think another PR beat you to this

Reply:
Conflicts with #1418

Reply:
Yeah I personally favor #1418 and #1419 over this. This one conflicts with both

Reply:
aight if that gets merged i'll remove from this